test/page-objects/ParticipantsPane.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Define participants pane elements.
+*/
+class ParticipantsPane {
+
+    // Participants pane object.
+    get ParticipantsPaneView() {
+        const participantsPane = $('.participants_pane');
```

Comment:
Please don't use jQuery, we want to kill it.

Reply:
<img width="845" alt="Screenshot 2022-03-16 at 15 41 19" src="https://user-images.githubusercontent.com/18324739/158603648-63209374-53e9-41da-a940-c606cffdf300.png">